jerry-main/main-unix.c

```diff
@@ -325,6 +292,71 @@ register_js_function (const char *name_p, /**< name of the function */
 } /* register_js_function */
 
 /**
+ * Command line option IDs
+ */
+typedef enum
+{
+  OPT_HELP,
+  OPT_VERSION,
+  OPT_MEM_STATS,
+  OPT_MEM_STATS_SEP,
+  OPT_PARSE_ONLY,
+  OPT_SHOW_OP,
+  OPT_SHOW_RE_OP,
+  OPT_DEBUG_SERVER,
+  OPT_SAVE_SNAP_GLOBAL,
+  OPT_SAVE_SNAP_EVAL,
+  OPT_SAVE_LIT_LIST,
+  OPT_SAVE_LIT_C,
+  OPT_EXEC_SNAP,
+  OPT_LOG_LEVEL,
+  OPT_ABORT_ON_FAIL,
+  OPT_NO_PROMPT
+} jerry_opt_id_t;
```

Comment:
Based on the previous conversation on the naming conventions, I'd rename this to `cli_opt_id_t`. `jerry_` prefix should be reserved for API functions and types.